x-pack/plugin/ent-search/src/test/java/org/elasticsearch/xpack/entsearch/engine/EngineIndexServiceTests.java

```diff
@@ -62,7 +63,7 @@ protected Collection<Class<? extends Plugin>> getPlugins() {
     }
 
     public void testCreateEngine() throws Exception {
-        final Engine engine = new Engine("my_engine", new String[] { "index_1" }, null);
+        final Engine engine = new Engine("my_engine", new String[] { "index_1" }, null, UPDATED_AT);
```

Comment:
We could check that the updated at result is greater than the time the test started, for example, so we don't have to provide updated at as part of the request